eclair-core/src/main/resources/reference.conf

```diff
@@ -461,7 +461,10 @@ eclair {
     max-per-peer-per-second = 10
 
     # Consider a message to be lost if we haven't received a reply after that amount of time
-    reply-timeout = 20 seconds
+    reply-timeout = 5 seconds
+
+    # If we expect a reply but do not get one, retry until we reach this number of attempts
+    max-attempts = 1
```

Comment:
Wouldn't it make more sense to use a default that actually does a retry, for example 3? Especially when offers provide multiple blinded paths, it would be a waste to only try one of them...